src/SumoCoders/FrameworkCoreBundle/Resources/assets/sass/components/_forms.scss

```diff
@@ -18,26 +18,26 @@ label abbr[title], label abbr[data-original-title] {
 }
 
 .form-control:focus {
-  @include box-shadow(none);
+  box-shadow: none;
   border: 2px solid #747e9f;
 }
 
 .form-control.input-sm {
-  @include font-size(12);
+  font-size: 12px;
```

Comment:
em's zijn gemakkelijk om alle typografie in 1 keer te schalen zonder elke unieke lettergrootte individueel te moeten instellen. Bootstrap gebruikt momenteel nog altijd pixels om de lettergroottes  in te stellen, en het mengen van em's en pixels kan onverwachte resultaten opleveren. Ik zou voorlopig dus met pixels blijven werken.